script.js

```diff
@@ -177,8 +177,36 @@ showPopupBtn.forEach((e, i) => {
 
 const form = document.querySelector('form');
 const email = document.getElementById('email');
+const fName = document.getElementById('name');
+const textArea = document.getElementById('textarea');
 const error = email.nextElementSibling;
 
+const formChangeEvents = [fName, email, textArea];
+formChangeEvents.forEach((eachField) => {
+  eachField.addEventListener('change', () => {
+    error.innerHTML = '';
+    const formDataObject = {
+      nameKey: fName.value,
```

Comment:
- [ ] I have noticed the name is not saved in the localStorage, so the name informs preserved in the browser. Kindly ensure that the name is saved so that when the browser is reloaded, the first name and last name details are preserved in the browser ☺️.
- [ ] 
![image](https://github.com/zhorabay/My-portfolio/assets/99394374/1949276b-b4e7-4cb8-a163-8ca3a9f4ef14)